### ISSUE TITLE
mosquitto bridge support not available

### ISSUE THREAD

**Issue by lautaro450** [overhandtech/microrouter-dockcross#1]
### Description ###
When we tried to configure mosquitto inside the microrouter as a bridge we received this warning message:

`Warning: Bridge support not available.`

### Acceptance criteria ###
* [x] Build mosquitto from scratch with bridge support
* [x] Write documentation about how to do it - see issue #2

### Screenshot ###
<img width="802" alt="Screen Shot 2019-05-16 at 7 48 19 PM" src="https://user-images.githubusercontent.com/15842049/57906280-0a0f6580-78ac-11e9-8d34-770f2bd512da.png">

**Comment by lautaro450** [comment]
**UPDATE** 

**mosquitto.conf**

~~~~
user mosquitto

port 1883
protocol mqtt

listener 9001
protocol websockets

#connection to AWS EC2 mosquitto
connection bridge-01
address 52.81.60.134:1883
topic # out 0
topic # in 0

log_type all
websockets_log_level 1023
~~~~

**Comment by rngadam** [comment]
working for me:

```
root@GL-USB150:~# ./mosquitto -c mosquitto.conf 
1558074222: mosquitto version 1.5.8 starting
1558074222: Config loaded from mosquitto.conf.
1558074222: Opening websockets listen socket on port 9001.
1558074222: Initial logging level 1023
1558074222: Libwebsockets version: 3.1.99 v3.1.0-116-g62c5a78
1558074222: IPV6 not compiled in
1558074222:  LWS_DEF_HEADER_LEN    : 4096
1558074222:  LWS_MAX_PROTOCOLS     : 5
1558074222:  LWS_MAX_SMP           : 1
1558074222:  sizeof (*info)        : 356
1558074222:  SYSTEM_RANDOM_FILEPATH: '/dev/urandom'
1558074222:  HTTP2 support         : available
1558074222: _realloc: size 520: context
1558074222: context created
1558074222: Using event loop: poll
1558074222:  default timeout (secs): 20
1558074222: _realloc: size 4096: pt_serv_buf
1558074222:  Threads: 1 each 1024 fds
1558074222:  mem: context:          4616 B (520 ctx + (1 thr x 4096))
1558074222:  mem: http hdr rsvd:   5169152 B (1 thr x (4096 + 952) x 1024))
1558074222: _realloc: size 8192: fds table
1558074222:  mem: pollfd map:       8192
1558074222: _realloc: size 4096: lws_lookup
1558074222:  mem: platform fd map:  4096 bytes
1558074222: _realloc: size 368: event pipe wsi
1558074222: lws_role_transition: 0x77ef5220: wsistate 0x200, ops pipe
1558074222: event pipe fd 5
1558074222: __insert_wsi_socket_into_fds: 0x77ef5220: tsi=0, sock=5, pos-in-fds=0
1558074222: _realloc: size 472: create vhost
1558074222: _realloc: size 112: vhost-specific plugin table
1558074222: _realloc: size 24: same vh list
1558074222: Creating Vhost 'default' port 9001, 3 protocols, IPv6 off
1558074222: _realloc: size 368: listen wsi
1558074222: _lws_vhost_init_server: lws_socket_bind says 9001
1558074222: lws_role_transition: 0x77ef5580: wsistate 0x200, ops listen
1558074222: lws_vhost_bind_wsi: vh default: count_bound_wsi 1
1558074222: __insert_wsi_socket_into_fds: 0x77ef5580: tsi=0, sock=7, pos-in-fds=1
1558074222:  Listening on port 9001
1558074222:  mem: per-conn:          368 bytes + protocol rx buf
1558074222:  canonical_hostname = GL-USB150
1558074222: lws_cancel_service
1558074222: Opening ipv4 listen socket on port 1883.
1558074222: Opening ipv6 listen socket on port 1883.
1558074222: Bridge local.GL-USB150.bridge-01 doing local SUBSCRIBE on topic #
1558074222: Connecting bridge bridge-01 (52.81.60.134:1883)
1558074222: Bridge GL-USB150.bridge-01 sending CONNECT
1558074222: lws_protocol_init
1558074222: Received CONNACK on connection local.GL-USB150.bridge-01.
1558074222: Bridge local.GL-USB150.bridge-01 sending UNSUBSCRIBE (Mid: 2, Topic: #)
1558074222: Bridge local.GL-USB150.bridge-01 sending SUBSCRIBE (Mid: 3, Topic: #, QoS: 0)
1558074222: Received PUBACK from local.GL-USB150.bridge-01 (Mid: 1)
1558074222: Received UNSUBACK from local.GL-USB150.bridge-01
1558074222: Received SUBACK from local.GL-USB150.bridge-01
1558074282: Sending PINGREQ to local.GL-USB150.bridge-01
1558074282: Received PINGRESP from local.GL-USB150.bridge-01
```

created branch with updated binary.

**Comment by rngadam** [comment]
my mosquitto source is at commit 2d360fd49521ef8c1694288023f93cc605697189

**Comment by rngadam** [comment]
@lautaro450 update doc

**Comment by lautaro450** [comment]
documentation was written and merged: #4
